src/main/proto/widget.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package smartthings;
+
+option java_package = "smartthings.ratpack.protobuf";
+option java_outer_classname = "WidgetProtos";
+
+message Widget {
```

Comment:
It works fine from the command line. Can't seem to get intellij to get test sources from the generated proto classes, though.